### ISSUE TITLE
Keyboard is not working within the visual studio code

### ISSUE THREAD

**Issue by ravibhushankumarsonu** [Microsoft/vscode#37775]
- VSCode Version: Code 1.17.2 (b813d12980308015bcd2b3a2f6efa5c810c33ba5, 2017-10-16T13:47:41.889Z)
- OS Version: Darwin x64 17.2.0
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
vscode-markdownlint|Dav|0.11.0
beautify|Hoo|1.1.1
latex-workshop|Jam|3.6.1
code-gnu-global|aus|0.2.2
jshint|dba|0.10.15
vscode-eslint|dba|1.4.3
python|don|0.7.0
csharpextensions|jch|1.3.0
MagicPython|mag|1.0.12
vscode-clang|mit|0.2.2
mssql|ms-|1.2.0
cpptools|ms-|0.14.0
csharp|ms-|1.12.1
java|red|0.13.0
python|tht|0.2.3
pdf|tom|0.2.0
vscode-java-debug|vsc|0.2.0
vscode-java-pack|vsc|0.2.0

---

Steps to Reproduce:

1. Whenever I started the visual studio It's stoped taking any keyboard input
2.

Reproduces without extensions: Yes/No

**Comment by kmelmon** [comment]
I'm seeing this as well just after installing the latest version (1.17.2).
This bug makes the entire product unusable - I cannot write any code!  PLEASE FIX THIS ASAP!!!

**Comment by prajapatiravi257** [comment]
I am also facing the same issue, my current system is 
Manjaro KDE 17.0.6 with visual studio code 1.17.2

**Comment by prajapatiravi257** [comment]
I am also facing the same issue, my current system is 
Manjaro KDE 17.0.6 with visual studio code 1.17.2

**Comment by michael-mayes** [comment]
Seeing this as well

Can delete text, but cannot enter text

Unaware of an update to the actual version (was on 1.17.2 before and after the issue)

Having to go back to Sublime Text for now

**Comment by michael-mayes** [comment]
Have tried toggling some extensions

All extensions disabled - no issue

All extensions enabled - issue

All extensions enabled except cpptools 0.14.0 - no issue

**Comment by michael-mayes** [comment]
Looking in Process Explorer, I saw that the Intellisense process was not running

Uninstalling the extension and reinstalling it has brought the Intellisense process back and restored the ability to type

**Comment by ejmvar** [comment]
Same here.
It was 1.17.2 before
Today, CPU usage goes high, but not for more than a minute.
I expected, after CPU usage became low again, the editor window should become responsive.
But:
- As mentioned, code editor does not receive keyboard input
- Some shortcuts work: comment line or block (as an example)

Steps I followed, whith no luck:
- removed configuration (~/.config/Code): clean restart fails the same way
- removed add-ons (one by one): still the same
- Uninstalled (full remove on LinuxMint) and reinstalled: same symptoms

I had to reinstall from 1.14 (a deb I had), and works ok!

I must, again, reference that 1.17.2 was working for, at least, a month.

Until now, cannot understand what made the difference between yesterday and today.

Cheers

**Comment by ejmvar** [comment]
Update:
Version 1.14 is not working anymore, just like 1.17.2.

How could we trace this back??

**Comment by ejmvar** [comment]
Update: maybe I'm somehow STUPID (maybe).

At some point, after repeating deletions from .config/Code, deinstallations, reinstallations and frustrations..

Code-1.16.1 whithout preferences is working (more than an hour, by now)
Issues:
- Line comment (Ctrl+Alt+7) on TS files fires : "command 'numberedBookmarks.toggleBookmark7' not found"

I remember code-1.17 failed with several shortcuts (that lead me to uninstalling add-ons).
Is this info useful?

I hope that.

**Comment by felipeferri** [comment]
I was having the exact same problem as @michael-mayes  commented. I couldn't edit any source file, but was able to delete characters. I am running VSCode 1.17.2 on macos 10.12.6.

Tried on several different workspaces, and everywhere had the same problem.

As @azzazkhan suggested, I disabled all extensions, restarted VSCode and it worked again. Then I enabled all the extensions again and it continued working.

**Comment by felipeferri** [comment]
Actually, when I enabled all extensions and restarted VSCode it stopped working again. Then I tried disabling the extensions one by one and restarting until it started working again.

I found out the culprit was the C/C++ extension (ms-vscode.cpptools). When I disable it VSCode works, when I enable it VSCode stops working.

**Comment by michael-mayes** [comment]
Same as I saw then

Seeing as I have now resolved it (for now) by reinstalling the CPP Tools extension, it would be handy if you could take a look at what Bob asked under https://github.com/Microsoft/vscode-cpptools/issues/1215 as I can't help further

**Comment by felipeferri** [comment]
@michael-mayes I didn't read in much detail all the answers and didn't notice you had proposed the correct workaround. Sorry. :-/

**Comment by michael-mayes** [comment]
No need to apologise, you did nothing wrong!

It would be handy if you could try the logging level change under that other call be reinstalling as I cannot now

**Comment by arindamgit** [comment]
I had Arduino extension. Deleting that worked

**Comment by alexandrudima** [comment]
Please update to latest stable. In 1.17 we had a bug where typing in the editor was blocked on activating emmet (a built-in extension). -- #36575

**Comment by dlbewley** [comment]
same issue with 1.18.0. i really want to like this editor! :(

**Comment by alexandrudima** [comment]
@dlbewley Please open a separate issue and include all the information requested in the issue template.

**Comment by shacall2** [comment]
Having same issue with 1.18.1.  So frustrating.  Such a great editor, when I can actually use it.

**Comment by bilalMlaik** [comment]
i am facing the same issue with 1.18.1.

**Comment by alexandrudima** [comment]
@shacall2 @bilalMlaik Please open separate issues and include all the information requested in the issue template. The "me too" comments are not helping anyone.
